### PR TITLE
Add PPP details

### PR DESCRIPTION
**Pull request by clydelhui**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -301,13 +302,27 @@ Examples:
 * `re_edit 1 1 d/Tuesday`
 * `re_edit 1 1 t/14:00`
 
+### Export a person: `export`
+Export a person's details from the address book. 
+
+Format: `export [INDEX_OF_PERSON]`
+
+* Exports a person's details with the specified `INDEX`
```
Comment (Alvozyn) [inline docs/UserGuide.md:310]: `INDEX` can be changed into `INDEX_OF_PERSON` (following the format) to make it clearer

File: docs/UserGuide.md [hunk 2]
```diff
@@ -301,13 +302,27 @@ Examples:
...
 
 Displays the time slots in a week when all members of the specified group are free (no events).
 
-Format: free GROUP_INDEX [START_DATE]
+Format: `free GROUP_INDEX [START_DATE]`
 
-* Finds unoccupied time slots within members of the group at the specified `INDEX`. The index refers to the index number
+* Finds unoccupied time slots within members of the group at the specified `GROUP_INDEX`. The index refers to the index number
   shown in the displayed group list. The index *must be a positive* 1, 2, 3 ...
 * The `START_DATE` field is optional. The time slots will be shown for the week ahead, starting from that date. If it is
```
Comment (Alvozyn) [inline docs/UserGuide.md:327]: Perhaps the format of `START_DATE` can be mentioned here `dd/MM/yyyy` cause the previous `START_DATE` consist of time

**Review by Alvozyn — COMMENTED** [review]
Lgtm. Just a minor change will do!

**Comment by lyndonlim27** [comment]
LGTM!

**Pull request merged**